### ISSUE TITLE
Inferencing data using RoBERTa finetuned model

### ISSUE THREAD

**Issue by chengfx** [pytorch/fairseq#981]
Hi, I encounter some problems of inferencing data using finetuned model when I am following the official instructions.

1.  **Loading model**

Below is my command,
 
```
roberta = RobertaModel.from_pretrained('checkpoints/lr1e-05_mp512', checkpoint_file='checkpoint_1_10.pt')
```
When I run this command, there are following errors
```
No such file or directory: '~/project/fairseq/checkpoints/lr1e-05_mp512/input0/dict.txt'
No such file or directory: '~/project/fairseq/checkpoints/lr1e-05_mp512/label/dict.txt'
```
I put the *dict.txt* into the folder *~/project/fairseq/checkpoints/lr1e-05_mp512/* like official pretrained model folder. Why is it correct for official pretrained model? Is there any difference between my model and official model?

2. **Dictionary** 

My task is the sentence pair classification. Do I need to prepare following folders for loading model?
```
~/project/fairseq/checkpoints/lr1e-05_mp512/input0/dict.txt'
~/project/fairseq/checkpoints/lr1e-05_mp512/input1/dict.txt'
~/project/fairseq/checkpoints/lr1e-05_mp512/label/dict.txt'
```

3. **The default name of classification_head**

I saw the predict function needs a name of classification head
```
prediction = roberta.predict('mnli', tokens).argmax().item()
```
What's the default name of the classification head? I just finetuned model using following command
```
python train.py <parameters>
```

4. **Inferencing data**
Could I encode entire data firstly and then inference it like training phase? Is there any instruction?

**Comment by ngoyal2707** [comment]
1) What command did you use for finetuning your model? If you followed: https://github.com/pytorch/fairseq/blob/master/examples/roberta/README.finetune_custom_classification.md then you should have the two folders `input0` and `label`.
and then you should be able to give path as following:
```
roberta = RobertaModel.from_pretrained('checkpoints/lr1e-05_mp512', checkpoint_file='checkpoint_1_10.pt', data_name_or_path=/path/to/data/folder/)
```
 
2) If you followed the readme as mentioned above, you should already have these.

3) It's ["sentence_classification_head"](https://github.com/pytorch/fairseq/blob/master/fairseq/tasks/sentence_prediction.py#L193)

4) yes, you can do that. Look at `checkpoint_utils.load_model_ensemble` function. If you need detailed code snipped, I might be able to write one. Lemme know first, if above 3 steps still doesn't work you.

**Comment by ngoyal2707** [comment]
I will close this issue. Feel free to ask if there are any further questions

**Comment by chengfx** [comment]
Thanks, it works now
